3rd-party-licenses/font-polyDisplaySemi-copyright-and-licence.rtf

```diff
@@ -0,0 +1,19 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2513
```

Comment:
Why is this format being used? There's a file for asset licenses, `asset-licenses.txt`. SIL is already there. You can probably just copy what's in there for Jost

Reply:
I just download it from the link here ([https://wiki.polypoly.eu/pages/viewpage.action?pageId=85624098](https://wiki.polypoly.eu/pages/viewpage.action?pageId=85624098)) and directly copied it. I'll change it to the standard format, thanks!